monitor.py

```diff
@@ -111,6 +114,9 @@ def __init__(self) -> None:
         self.systempopulation: Optional[int] = None
         self.started: Optional[int] = None  # Timestamp of the LoadGame event
 
+        self._navroute_retries = 0
```

Comment:
`# Retries currently remaining`

This calls out that's not the 'opposite' of "retries already attempted".